WalletWasabi/BitcoinCore/Configuration/Whitening/WhiteList.cs

```diff
@@ -1,10 +1,11 @@
 using NBitcoin;
+using System.Diagnostics.CodeAnalysis;
 
 namespace WalletWasabi.BitcoinCore.Configuration.Whitening
 {
 	public class WhiteList : WhiteEntry
 	{
-		public static bool TryParse(string value, Network network, out WhiteList white)
+		public static bool TryParse(string value, Network network, [NotNullWhen(true)] out WhiteList? white)
```

Comment:
The method seems to be unused.

Reply:
Yes, should we remove the whole class then? In a follow up PR?

Reply:
Makes sense to me.